libcontainer/nsenter/nsexec.c

```diff
@@ -597,8 +651,8 @@ void nsexec(void)
 						update_setgroups(child, SETGROUPS_DENY);
 
 					/* Set up mappings. */
-					update_uidmap(child, config.uidmap, config.uidmap_len);
-					update_gidmap(child, config.gidmap, config.gidmap_len);
+					update_uidmap(child, config.is_rootless, config.uidmap, config.uidmap_len);
+					update_gidmap(child, config.is_rootless, config.gidmap, config.gidmap_len);
```

Comment:
I am using `newuidmap` and `newgidmap` only after an `EPERM`.  When running as `root` that should never happen